doc/source/whatsnew/v1.4.0.rst

```diff
@@ -186,9 +186,45 @@ Now the float-dtype is respected. Since the common dtype for these DataFrames is
 
     res
 
-.. _whatsnew_140.notable_bug_fixes.notable_bug_fix3:
+.. _whatsnew_140.notable_bug_fixes.write_compliant_parquet_nested_type:
 
-notable_bug_fix3
+Write compliant Parquet nested types if possible
```

Comment:
you don't need to add a whole sub-section on this. a single line note is fine.

Reply:
I just figured that since it could break people's stuff it was worth calling out but shall remove if you'd prefer?

Reply:
the entire poitn is this *wont'* break anyone

Reply:
Do you want to expand on this? This change will change the data that Pandas outputs. If a user is expecting and handling the current output (not necessarily back into Pandas as Parquet is a cross platform format) then this could break their code elsewhere? 

Am I misunderstanding your point?

Reply:
just a single line is fine here